src/mon/Monitor.cc

```diff
@@ -950,6 +950,12 @@ int Monitor::init()
   mgrmon()->prime_mgr_client();
 
   state = STATE_PROBING;
+
+  if (!elector.peer_tracker_is_clean()){
+    dout(10) << "peer_tracker is not clean, clearing ..." << dendl;
```

Comment:
Let's call this something besides "clean". Could say "consistent", or "peer_tracker looks inconsistent due to previous bad update logic; clearing..."

Reply:
Addressed in new push